pkg/apis/kops/dockerconfig.go

```diff
@@ -20,6 +20,8 @@ package kops
 type DockerConfig struct {
 	// AuthorizationPlugins is a list of authorization plugins
 	AuthorizationPlugins []string `json:"authorizationPlugins,omitempty" flag:"authorization-plugin,repeat"`
+	// AddRuntime registers an additional OCI compatible runtime (default [])
+	Runtimes []string `json:"addRuntime,omitempty" flag:"add-runtime,repeat"`
```

Comment:
Also please move down to order the flags alphabetically.
```suggestion
	// Runtimes registers an additional OCI compatible runtime (default [])
	Runtimes []string `json:"runtimes,omitempty" flag:"add-runtime,repeat"`
```

Reply:
on it. was a miss